Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -167,6 +167,19 @@ public PythonProjectNode(IServiceProvider serviceProvider)
             }
         }
 
+        protected internal override void SetCurrentConfiguration() {
+            base.SetCurrentConfiguration();
+
+            if (!IsProjectOpened)
+                return;
+
+            if (this.IsAppxPackageableProject()) {
+                EnvDTE.Project automationObject = GetAutomationObject() as EnvDTE.Project;

```

Comment:
Don't do a safe cast unless you're going to check the result. InvalidCastExceptions are more helpful than NullReferenceExceptions. (And if this code came from elsewhere, let's take the opportunity to fix it now.)